mkosi/__init__.py

```diff
@@ -239,67 +239,39 @@ def configure_os_release(state: MkosiState) -> None:
         newosrelease.rename(osrelease)
 
 
-def configure_autologin(state: MkosiState) -> None:
-    if not state.config.autologin:
-        return
-
-    with complete_step("Setting up autologin…"):
-        dropin = state.root / "usr/lib/systemd/system/console-getty.service.d/autologin.conf"
-        with umask(~0o755):
-            dropin.parent.mkdir(parents=True, exist_ok=True)
-        with umask(~0o644):
-            dropin.write_text(
-                """\
+def configure_autologin_service(state: MkosiState, service: str, extra: str) -> None:
+    dropin = state.root / f"usr/lib/systemd/system/{service}.d/autologin.conf"
+    with umask(~0o755):
+        dropin.parent.mkdir(parents=True, exist_ok=True)
+    with umask(~0o644):
+        dropin.write_text(
+            textwrap.dedent(
+                f"""\
                 [Service]
                 ExecStart=
-                ExecStart=-/sbin/agetty -o '-f -p -- \\\\u' --autologin root --noclear --keep-baud console 115200,38400,9600 $TERM
+                ExecStart=-agetty -o '-f -p -- \\\\u' --autologin root {extra} $TERM
                 StandardInput=tty
                 StandardOutput=tty
                 """  # noqa: E501
```

Comment:
That `noqa` shouldn't be necessary any longer, since the `EcecStart` is shorter now.

Reply:
I dropped `noqa`.